oam/etc/Columnstore.xml

```diff
@@ -96,6 +96,7 @@
 		<!-- <MediumPriorityPercentage>30</MediumPriorityPercentage> -->
 		<!-- <LowPriorityPercentage>10</LowPriorityPercentage> -->
 		<DirectIO>y</DirectIO>
+        <MaxBPPSendQueue>250M</MaxBPPSendQueue>
```

Comment:
As you know we can't deliver Columnstore.xml changes until we deliver the MCOL-4928.

Reply:
This is the default, so we can leave it out of the xml for now.